lib/taste_tester/commands.rb

```diff
@@ -388,7 +393,7 @@ def self._build_dependency_tree(deps)
 
     def self._print_impact(final_impact)
       if TasteTester::Config.json
-        puts "JSON output not yet supported"
+        puts 'JSON output not yet supported'
```

Comment:
This should return or exit with an error... and in fact in main when you do option parsing, if you're not gonna support JSON for impact, check if those two options were specified together and exit there before you do all this other work.

Reply:
That was more of a placeholder for testing. I will replace this with an exit() if I don't get around to it before merging.

Reply:
This has been replaced with an `exit(1)` at the beginning of `impact`.  I am going to leave it as a TODO/"exercise for the reader" for now unless you feel strongly about it.